.bazelrc

```diff
@@ -20,4 +20,4 @@ try-import ./.bazel-remote-cache.rc
 build --incompatible_strict_action_env --java_language_version=11 --javacopt='--release 11' --@io_bazel_rules_docker//transitions:enable=false
 run --incompatible_strict_action_env
 test --incompatible_strict_action_env --test_env=PATH
-
+--java_runtime_version=remotejdk_11
```

Comment:
I don't think this line was part of the ARM build fix. Please verify and remove.

Reply:
I think we actually want this :) let's keep it

Reply:
Is this line alone valid syntax, or does it have to be:
`common --java_runtime_version=remotejdk_11`